stonesoup/predictor/kalman.py

```diff
@@ -326,17 +329,19 @@ def _transition_and_control_function(self, prior_state, **kwargs):
             control
         """
 
-        return (self.transition_model.function(prior_state, **kwargs)
-                + self.control_model.control_input())
+        return self.transition_model.function(prior_state, **kwargs) + \
+            self.control_model.function(**kwargs)
 
     @predict_lru_cache()
-    def predict(self, prior, timestamp=None, **kwargs):
+    def predict(self, prior, control_input=None, timestamp=None, **kwargs):
         r"""The unscented version of the predict step
 
         Parameters
         ----------
         prior : :class:`~.State`
             Prior state, :math:`\mathbf{x}_{k-1}`
+        control_input: :class:`~.State`
+            Control input vector, :math:`\mathbf{u}_k`
         timestamp : :class:`datetime.datetime`
             Time to transit to (:math:`k`)
```

Comment:
Minor change to be consistent with other `predict` methods argument orders.

```suggestion
    def predict(self, prior, timestamp=None, control_input=None, **kwargs):
        r"""The unscented version of the predict step

        Parameters
        ----------
        prior : :class:`~.State`
            Prior state, :math:`\mathbf{x}_{k-1}`
        timestamp : :class:`datetime.datetime`
            Time to transit to (:math:`k`)
        control_input: :class:`~.State`
            Control input vector, :math:`\mathbf{u}_k`
```